src/style.css

```diff
@@ -0,0 +1,105 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
```

Comment:
do you need this one? I believe that on a design there is only 400, and 900

Reply:
I thought this is for this
### Requirements:
...
- default `font-weight` must be 300.

Reply:
you did it in body styles already